images/dcos/bootstrap.sh

```diff
@@ -84,23 +79,19 @@ echo "* service proxy: $service_proxy"
 etcd_server_port=${ETCD_SERVER_PORT:-4001}
 etcd_server_peer_port=${ETCD_SERVER_PEER_PORT:-4002}
 
-ENABLE_ETCD_SERVER=${ENABLE_ETCD_SERVER:-false}
-if [ "${ENABLE_ETCD_SERVER}" != true ]; then
-  etcd_server_list=${ETCD_SERVER_LIST:-http://${service_proxy}:${etcd_server_port}}
-else
-  etcd_advertise_server_host=${ETCD_ADVERTISE_SERVER_HOST:-127.0.0.1}
-  etcd_server_host=${ETCD_SERVER_HOST:-127.0.0.1}
+ETCD_MESOS=${ETCD_MESOS:-disabled}
```

Comment:
sttts and I settled on ETCD_MESOS_FRAMEWORK_NAME

On Wed, Oct 14, 2015 at 11:17 AM, Dr. Stefan Schimanski <
notifications@github.com> wrote:

> In images/dcos/bootstrap.sh
> <https://github.com/mesosphere/kubernetes-mesos/pull/556#discussion_r42007387>
> :
>
> > @@ -84,23 +79,19 @@ echo "* service proxy: $service_proxy"
> >  etcd_server_port=${ETCD_SERVER_PORT:-4001}
> >  etcd_server_peer_port=${ETCD_SERVER_PEER_PORT:-4002}
> >
> > -ENABLE_ETCD_SERVER=${ENABLE_ETCD_SERVER:-false}
> > -if [ "${ENABLE_ETCD_SERVER}" != true ]; then
> > -  etcd_server_list=${ETCD_SERVER_LIST:-http://${service_proxy}:${etcd_server_port}}
> > -else
> > -  etcd_advertise_server_host=${ETCD_ADVERTISE_SERVER_HOST:-127.0.0.1}
> > -  etcd_server_host=${ETCD_SERVER_HOST:-127.0.0.1}
> > +ETCD_MESOS=${ETCD_MESOS:-disabled}
>
> I think you miss my point. I don't argue against the variable, just
> against the bad name, which is neither a verb nor an adjective, but only
> two context-less words.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/kubernetes-mesos/pull/556/files#r42007387>.
>